polaris/ocean/__init__.py

```diff
@@ -30,11 +30,11 @@ def __init__(self):
 
         # single column
         add_single_column_tasks(component=self)
-        add_sphere_transport_tasks(component=self)
 
         # spherical: please keep these in alphabetical order
         add_cosine_bell_tasks(component=self)
         add_geostrophic_tasks(component=self)
+        add_sphere_transport_tasks(component=self)
```

Comment:
We want to organize these tasks roughly alphabetically first by `planar`, `single_column`, `spherical` and then by specific subdirectory.  Since this isn't easy to do with the 4 types of tasks in spherical transport, we'll put it at the end.